package.json

```diff
@@ -84,6 +84,7 @@
         "@react-hookz/web": "^12.0.0",
         "classnames": "^2.2.6",
         "react-modal": "^3.15.1",
+        "resize-observer-polyfill": "^1.5.1",
```

Comment:
I've noticed that you've downgraded the `package-lock.json` file to version 1. Please use the right npm version (by running `nvm use` in the project folder).